docs/architecture/solana/migration-faq.mdx

```diff
@@ -54,8 +52,8 @@ Here are 2 ways to do this:
 
 :::warning
 
-Treasuries for each Network are very low right now. However, better rates for swapping tokens may be
-available using other services.
+Always check the swap or exchange rate before confirming a transaction. You are responsible for the
+confirmation. The default floor rate may not be the best exchange rate available.
```

Comment:
in-app swaps now leverage Jupiter, and are not locked to the treasury swap rate. This was not the case immediately post-migration